src/test/java/uk/gov/hmcts/reform/managecase/service/NoticeOfChangeServiceTest.java

```diff
@@ -0,0 +1,410 @@
+            CaseViewResource caseViewResource = new CaseViewResource();
+            caseViewResource.setCaseViewActionableEvents(new CaseViewActionableEvent[]{caseViewActionableEvent});
+            CaseViewType caseViewType = new CaseViewType();
+            caseViewType.setName(CASE_TYPE_ID);
+            caseViewType.setId(CASE_TYPE_ID);
+            CaseViewJurisdiction caseViewJurisdiction = new CaseViewJurisdiction();
+            caseViewJurisdiction.setId(JURISDICTION);
+            caseViewType.setJurisdiction(caseViewJurisdiction);
+            caseViewResource.setCaseType(caseViewType);
+            given(dataStoreRepository.findCaseByCaseId(CASE_ID))
+                .willReturn(caseViewResource);
+
+            // Internal ES query
+            SearchResultViewHeader searchResultViewHeader = new SearchResultViewHeader();
+            FieldTypeDefinition fieldTypeDefinition = new FieldTypeDefinition();
+            fieldTypeDefinition.setType(PREDEFINED_COMPLEX_CHANGE_ORGANISATION_REQUEST);
+            fieldTypeDefinition.setId(PREDEFINED_COMPLEX_CHANGE_ORGANISATION_REQUEST);
+            searchResultViewHeader.setCaseFieldTypeDefinition(fieldTypeDefinition);
+            searchResultViewHeader.setCaseFieldId(CHANGE_ORG);
+            caseFields.put(DATE_FIELD, new TextNode("2020-10-01"));
+            caseFields.put(DATETIME_FIELD, new TextNode("1985-12-30"));
+            caseFields.put(TEXT_FIELD, new TextNode("Text Value"));
+            ObjectMapper mapper = new ObjectMapper();
+            JsonNode actualObj = mapper.readValue("{\n"
+                                                      + "  \"OrganisationPolicy1\": {\n"
+                                                      + "    \"OrgPolicyCaseAssignedRole\": \"Applicant\",\n"
+                                                      + "    \"OrgPolicyReference\": \"Reference\",\n"
+                                                      + "    \"Organisation\": {\n"
+                                                      + "      \"OrganisationID\": \"QUK822N\",\n"
+                                                      + "      \"OrganisationName\": \"CCD Solicitors Limited\"\n"
+                                                      + "    }\n"
+                                                      + "  }\n"
+                                                      + "}", JsonNode.class);
+
+            caseFields.put(PREDEFINED_COMPLEX_ORGANISATION_POLICY, actualObj);
+            SearchResultViewItem item = new SearchResultViewItem("CaseId", caseFields, caseFields);
+            viewItems.add(item);
+            SearchResultViewHeaderGroup correctHeader = new SearchResultViewHeaderGroup(
+                new HeaderGroupMetadata(JURISDICTION, CASE_TYPE),
+                Arrays.asList(searchResultViewHeader), Arrays.asList("111", "222")
+            );
+            List<SearchResultViewHeaderGroup> headers = new ArrayList<>();
```

Comment:
`List<SearchResultViewHeaderGroup> headers = List.of(correctHeader);` ?

Reply:
done